website/blog/2022-07-28-srb-tapioca.md

```diff
@@ -0,0 +1,266 @@
+if you need to reference it.
+
+If you'd like to start using Tapioca in a project that already uses `srb rbi`,
+[see this migration guide](#TODO).
+
+With the announcements out of the way, let's dive into some questions we imagine
+you might have.
+
+## What is Tapioca?
+
+[Tapioca](https://github.com/Shopify/tapioca) is a Ruby gem that provides
+command line tools for generating [RBI files](/docs/rbi). From its
+documentation, "Tapioca surfaces types and methods from many sources that Sorbet
+cannot otherwise see—such as gems, Rails and other DSLs—compiles them into RBI
+files, and makes it easy for you to add gradual typing to your application."
+
+This is largely what the `srb init` and `srb rbi` commands, (which are provided
+by Sorbet itself) already do, but Tapioca does it better. We can explain what
+Tapioca does by way of comparison:
+
+| `srb`                        | `tapioca`                                         |
+| ---------------------------- | ------------------------------------------------- |
+| `srb init`                   | [`tapioca init`][tapioca-init]                    |
+| `srb rbi config`             | [`tapioca config`][tapioca-config]                |
+| `srb rbi sorbet-typed`       | [`tapioca annotations`][tapioca-annotations]      |
+| `srb rbi gems`               | [`tapioca gems`][tapioca-gems]                    |
+| `srb rbi hidden-definitions` | Partially covered by [`tapioca dsl`][tapioca-dsl] |
+| `srb rbi todo`               | [`tapioca todo`][tapioca-todo]                    |
+| `srb rbi suggest-typed`      | Delegated to [`spoom bump`][spoom-bump]           |
+| `srb rbi find-gem-rbis`      | Folded inside `tapioca gems`                      |
+
+[tapioca-init]: https://github.com/Shopify/tapioca#getting-started
+[tapioca-config]: https://github.com/Shopify/tapioca#getting-started
+[tapioca-annotations]:
+  https://github.com/Shopify/tapioca#pulling-rbi-annotations-from-remote-sources
+[tapioca-gems]: https://github.com/Shopify/tapioca#generating-rbi-files-for-gems
+[tapioca-dsl]:
+  https://github.com/Shopify/tapioca#generating-rbi-files-for-rails-and-other-dsls
+[tapioca-todo]:
+  https://github.com/Shopify/tapioca#rbi-files-for-missing-constants-and-methods
+[spoom-bump]: https://github.com/Shopify/spoom#change-the-sigil-used-in-files
+
+You'll notice that most `srb` subcommands have a nearly (if not exactly)
+equivalent command in `tapioca`, which brings us to the next question.
+
+## What's different between `srb rbi` and `tapioca`?
+
+From one standpoint, they're quite similar: they both use various heuristics to
+generate RBI files before `srb tc` runs on the project. Specifically, they both
+rely on Sorbet to typecheck the project itself (Tapioca is not some sort of
+alternative type checker for Ruby).
+
+Both Tapioca and `srb rbi` generate RBIs for a project's gems and support
+various generating RBI files that teach Sorbet about various kinds of
```

Comment:
"support various generating RBI files" sounds weird. Should it be "support generating various RBI files"?

Reply:
good catch thanks.